docs/src/main/asciidoc/security-basic-authentication-howto.adoc

```diff
@@ -15,23 +15,39 @@ Enable xref:security-basic-authentication.adoc[Basic authentication] for your Qu
 
 == Prerequisites
 
-* You have installed at least one extension that provides an `IdentityProvider` based on username and password, such as xref:security-jdbc.adoc[Elytron JDBC].
+* You have installed at least one extension that provides an `IdentityProvider` based on username and password.
+For example:
+
+** xref:security-basic-authentication-tutorialsecurity-jpa.adoc[Quarkus Security Jakarta Persistence extensions (`security-jpa` or `security-jpa-reactive`)]
```

Comment:
This link is probably broken. It should be IMO `xref:security-jpa.adoc`

Reply:
The link is not broken. Question is, if the author wants to have that rendered Name for the xref link.

Reply:
@sberyozkin , @jsmrcka , @rsvoboda , @MichalMaler - Thanks all for reviewing :green_heart: 

I fixed the broken link! I couldn't decide whether the tutorial or concept for JPA worked best so I accidently combined both. Apologies for the churn. It is now fixed. Thanks!